app/models/user.rb

```diff
@@ -150,24 +150,25 @@ def grant_permission(supported_permission)
     application_permissions.where(supported_permission_id: supported_permission.id).first_or_create!
   end
 
-  # override Devise::Recoverable behavior to:
-  # 1. notify suspended users that they can't reset their password, and
-  # 2. handle emails blacklisted by AWS such that we conceal whether
-  #    or not an account exists for that email. moved from:
-  #    https://github.com/alphagov/signon/commit/451b89d9
+  # Override Devise::Recoverable behavior to notify suspended users that they
```

Comment:
Should this be 'notify unusable users'?

Reply:
Removed a bit, it's already clear from the code what it does.